3520/1/top_gems.rb

```diff
@@ -0,0 +1,4 @@
+require './lib/topgems.rb'
+require './lib/tg_parse.rb'
+
+TopGems.new.run(TGParse.parse(ARGV))
```

Comment:
looks ugly, try to make it under the hood, I mean passing params in such way

Reply:
> looks ugly, try to make it under the hood, I mean passing params in such way
Done.